### DISCUSSION TITLE
ANSI-SQL conversion

### DISCUSSION THREAD

**Discussion by KevinWellsMLC** [dbeaver/dbeaver#10629] [Q&A] [answered]
#### System information: 
- Windows 2012 R2
- DBeaver 7.2.4

#### Connection specification:
- Database Oracle  11g

Does DBeaver 7.2.4 have any functionality / tools to convert queries with Non-ANSI syntax to ANSI syntax in Oracle?

Thank you.

**Comment by kseniiaguzeeva** [comment, marked as answer]
Unfortunately DBeaver doesn't have such option.
